src/Component/Header/Header.jsx

```diff
@@ -55,7 +55,7 @@ function Header() {
     setToggleMenu((prevState) => !prevState);
   }
 
-  const menu = (
+  const menu = userInfo && userInfo.role === 2 ? (
```

Comment:
Những thứ liên quan như này thì tạo 1 constant cho nó
```suggestion
  const menu = userInfo && userInfo.role === ADMIN_ROLE ? (
```
file constants.js
```
const ADMIN_ROLE = 2
```